src/js/components/Anchor/__tests__/Anchor-test.tsx

```diff
@@ -220,4 +221,47 @@ describe('Anchor', () => {
     expect(getByLabelText('test-2')).toBeTruthy();
     expect(container).toMatchSnapshot();
   });
+
+  test('renders size specific theming', () => {
+    const theme = {
+      anchor: {
+        color: 'text-strong',
+        textDecoration: 'underline',
+        fontWeight: 700,
+        size: {
+          large: {
+            color: 'brand',
+            textDecoration: 'none',
+          },
```

Comment:
Include a fontWeight property since it is on the type definition in base.d.ts?

```suggestion
          large: {
            color: 'brand',
            fontWeight: 'bold',
            textDecoration: 'none',
          },
```